model/enduser/registry.yaml

```diff
@@ -0,0 +1,41 @@
+groups:
+  - id: registry.enduser
+    type: attribute_group
+    display_name: End User Attributes
+    brief: >
+      Describes information about the end user, which can be used as a subdomain of browser, client, or user domains.
+    attributes:
+      - id: enduser.id
+        type: string
+        deprecated: Replaced by `enduser.pseudo.id` attribute.
```

Comment:
deprecated attributes should be moved from registry to the deprecated section

Reply:
my understanding is that if the namespace is deprecated, then it will move everything to the deprecated folder. 
if some of the attributes are deprecated, then simply use `deprecated:`. please correct me if I'm wrong. an example will be helpful.